config/backends.go

```diff
@@ -51,39 +55,50 @@ type Aerospike struct {
 	Namespace  string `mapstructure:"namespace"`
 }
 
-func (cfg *Aerospike) validateAndLog() {
+func (cfg *Aerospike) validateAndLog() error {
+	if len(cfg.Host) < 1 {
+		return fmt.Errorf("Cannot connect to empty Aerospike host")
+	}
+	if cfg.Port <= 0 {
+		return fmt.Errorf("Cannot connect to Aerospike host at port %d", cfg.Port)
+	}
 	log.Infof("config.backend.aerospike.default_ttl_seconds: %d", cfg.DefaultTTL)
 	log.Infof("config.backend.aerospike.host: %s", cfg.Host)
 	log.Infof("config.backend.aerospike.port: %d", cfg.Port)
 	log.Infof("config.backend.aerospike.namespace: %s", cfg.Namespace)
+
+	return nil
 }
 
 type Azure struct {
 	Account string `mapstructure:"account"`
 	Key     string `mapstructure:"key"`
 }
 
-func (cfg *Azure) validateAndLog() {
+func (cfg *Azure) validateAndLog() error {
```

Comment:
I realize that this is not your PR change but shouldn't we also add config validations for these other backends?

Reply:
Yes, I'll code those on a separate PR. Almost couldn't resist the temptation of doing them on this one but it'd bloat this PR a lot more than needed and it'd be better and easier to review if we do it separately